src/components/tree-view/tree-view.st.css

```diff
@@ -1,51 +1,11 @@
-:import{
-    -st-from: "../../style/default-theme/variables.st.css";
-    -st-named: color_treeFocused, color_treeSelected;
-}
-
 .root {
```

Comment:
to properly be able to EXTEND the tree-node style (see radio-group for the correct pattern):
1. this file should contain an import of the `tree-node.st.css`.  
2. tree-node class should have `-st-extends: ...`

Reply:
done